addons/survey/controllers/main.py

```diff
@@ -364,20 +374,30 @@ def survey_display_page(self, survey_token, answer_token, **post):
         return request.render('survey.survey_page_fill',
             self._prepare_survey_data(access_data['survey_sudo'], answer_sudo, **post))
 
-    @http.route('/survey/get_background_image/<string:survey_token>/<string:answer_token>', type='http', auth="public", website=True, sitemap=False)
-    def survey_get_background(self, survey_token, answer_token):
-        access_data = self._get_access_data(survey_token, answer_token, ensure_token=True)
-        if access_data['validity_code'] is not True:
-            return werkzeug.exceptions.Forbidden()
-
-        survey_sudo, answer_sudo = access_data['survey_sudo'], access_data['answer_sudo']
-
+    def _get_background_image(self, model_name, res_id):
```

Comment:
Should this utility method be sorted elsewhere?
With other utility methods maybe?